Add scripts to make it easier for contributors to keep codestyle consistent

batmat opened the pull request:
I didn't really want to adjust manually the formatting on my previous PR #6 :smile:, so I added those scripts to make it easier for contributors.

This way, people can just type `npm run eslint-fix` and be sure the code is formatted as the existing .eslintrc is defining the style.

batmat committed:
eslint and eslint-fix scripts to keep codestyle consistent

I didn't really want to adjust manually the formatting on my previous PR,
so I added those scripts to make it easier for contributors.

This way, people can just type `npm run eslint-fix` and be sure the code
is formatted as the existing .eslintrc is defining the style.

batmat committed:
`npm run eslint-fix` applied

void666 commented:
`gulp fix` is already configured to do the same.`gulp fix`followed by `gulp check` should do this formatting.

batmat committed:
Use already configured gulp instead of eslint

batmat commented:
@void666 sorry for the latency, I just changed the PR to use gulp instead. Also changed the script names to `lint-check` and `lint-fix` so that it's more generic, than surfacing the formatting tool in use.
